apps/molgenis-components/src/components/tables/AggregateTable.vue

```diff
@@ -95,11 +93,16 @@ import Client from "../../client/client";
 import InputSelect from "../forms/InputSelect.vue";
 import { IColumn } from "../../Interfaces/IColumn";
 import { convertToCamelCase } from "../utils";
+import { INewClient } from "../../client/IClient";
 
 export default defineComponent({
   name: "AggregateTable",
   components: { TableStickyHeaders, InputSelect },
   props: {
+    canView: {
+      type: Boolean,
+      required: true,
+    },
```

Comment:
Instead of making it required, we could also choose to make it default to false,

Reply:
I thought this might help user of the component to understand what is going on, easing debugging of their setup. Do you agree?